client/README.md

```diff
@@ -1,208 +1,389 @@
-# Python Exemplary Client
-
-This is an exemplary command-line tool to simplify the process of running a
-Batch SQL Translation job using the Google Cloud Bigquery Migration Python
-client package.
-
-## Supported Python Versions 
-Python >= 3.7
-
-## Installation
-
-Clone or download this repository and install the exemplary Python client: 
-
+# BigQuery Migration Service Batch SQL Translator CLI
+
+The [BigQuery Migration Service][BQMS] includes a product known as the
+[batch SQL translator][batch SQL translator] that supports translation of a 
+wide variety of SQL dialects to Google Standard SQL. This command line tool
+simplifies the process of using the batch SQL translator by providing a
+framework for pre and postprocessing of SQL files as well as upload/download of
+SQL files to/from GCS.
+
+- [Quickstart](#quickstart)
+- [Installation](#installation)
+- [Basic Usage](#basic-usage)
+  - [Environment Variables](#environment-variables)
+  - [config.yaml](#configyaml)
+  - [Recommended Usage](#recommended-usage)
+- [Metadata Lookup](#metadata-lookup)
+    - [DDL and Metadata Dump](#ddl-and-metadata-dump)
+    - [Unqualified References](#unqualified-references)
+- [Pre and Postprocessing](#pre-and-postprocessing)
+  - [Handling Macro/Templating Languages](#handling-macrotemplating-languages)
+  - [Custom Pre and Postprocessing Logic](#custom-pre-and-postprocessing-logic)
+- [Renaming SQL Identifiers](#renaming-sql-identifiers)
+- [Deploying to Cloud Run](#deploying-to-cloud-run)
+
+## Quickstart
+
+```shell
+# Install the CLI as described in the Installation section below.
```

Comment:
Do we want to move the installation section here so users don't need to navigate back and forth?

Reply:
Yea I can put the installation section first.